skrub/_reporting/tests/test_table_report.py

```diff
@@ -123,6 +127,42 @@ def test_duration(df_module):
     assert re.search(r"2(\.0)?\s+days", TableReport(df).html())
 
 
+@pytest.mark.parametrize("filename_type", ["str", "Path", "file_object"])
+def test_write_html(pd_module, filename_type):
+    df = pd_module.make_dataframe({"a": [1, 2], "b": [3, 4]})
+    report = TableReport(df)
+
+    with TemporaryDirectory() as td:
```

Comment:
maybe we could use the pytest [`tmp_path`](https://docs.pytest.org/en/stable/how-to/tmp_path.html#the-tmp-path-fixture) fixture?